platform/SDL2Desktop/source/SDL2Host.cpp

```diff
@@ -77,7 +85,31 @@ InputState_t Host::scanInput(){
 
     while (SDL_PollEvent(&event)) {
         switch (event.type) {
+			
+			#if KB_ENABLED
+			case SDL_TEXTINPUT:
+				//Logger_Write( charset::upper_to_emoji(event.text.text).c_str() );
+				//Logger_Write("\n");
+				currKBKey = charset::upper_to_emoji(event.text.text);
+				currKBDown = true;
```

Comment:
these need to be reset before this event loop, but I'll do that after I merge the PR